inference-engine/src/gna_plugin/backend/dnn_components.hpp

```diff
@@ -12,11 +12,21 @@
 
 namespace GNAPluginNS {
 namespace backend {
+struct DnnComponentExtra {
```

Comment:
@admitriev-gna  please take a look - you'll need rebase you PR (https://github.com/openvinotoolkit/openvino/pull/2117) for delayed memory , and looks only tests will be added - rest done in current PR